2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,74 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+let recipe1 = {
+  title: "Pasta",
+  servings: 4,
+  ingredients: ["Flour","Eggs","Salt","Olive Oil","Water","Tomato"],
+  printRecipe: function(){
+    console.log(this.title);
+    console.log(`Serves: ${this.servings}`);
+    console.log("Ingredients")
+    for (const ingredient of this.ingredients) {
```

Comment:
I really like your use of a for loop here to log out each value from the ingredients key :)